.tool-versions

```diff
@@ -0,0 +1 @@
+nodejs 12.20.0
```

Comment:
Is this intentionally added? If so, seems out of scope for this PR - if not, please revert :)

Reply:
removed. I think we should consider adding this, but you are right - it's out of scope here